pkg/jsonedit/rename.go

```diff
@@ -47,11 +47,18 @@ func Rename(jsonBytes []byte, paths map[string]string) ([]byte, error) {
 			items := []interface{}{}
 
 			for _, child := range parentList {
-				childBytes, err := json.Marshal(child)
-				renamedBytes, err := Rename(childBytes, renamePaths)
-
+				var childBytes []byte
+				childBytes, err = json.Marshal(child)
+				if err != nil {
+					return nil, err
```

Comment:
As in other parts of the code, I think this also should not return but most likely just continue